lib/code_corps/emails/forgot_password_email.ex

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Emails.ForgotPasswordEmail do
+  import Bamboo.Email
+  import Bamboo.PostmarkHelper
+
+  alias CodeCorps.Emails.BaseEmail
+
+  def create(user) do
+    BaseEmail.create
+    |> to(user.email)
+    |> template(template_id(), {})
+  end
+
+  defp template_id, do: Application.get_env(:code_corps, :postmark_receipt_template)
```

Comment:
Make it a `:postmark_forgot_password_template`. 

We still need to add the actual template, though, but that's not something you can create. Just assume it's there, under the above configuration key. 

@JoshSmith, you created the receipt template, so I'm referring to you to actually create the template with that ID and add it to the environment.